se agrego el endpoint para crear usuarios

susana-vargas opened the pull request.

susana-vargas committed:
se agrego el endpoint para crear usuarios

Cruz-Flores commented on src/users/dto/user.dto.ts:
```diff
@@ -0,0 +1,22 @@
+
+export class CreateUserDto {
+
+  id: string
```
faltan los decoradores de validacion

Cruz-Flores commented on src/users/user.contoller.ts:
```diff
@@ -0,0 +1,17 @@
+
+
+  @Post()
+  createUser(@Body(ValidationPipe) newUser: CreateUserDto){
```
el body no lleva el validation pipe

Cruz-Flores commented on src/users/dto/user.dto.ts:
```diff
@@ -0,0 +1,22 @@
+
+  id: string
+
+  @IsInt()
```
es IsNumber no IsInt

Cruz-Flores commented on src/users/user.service.ts:
```diff
@@ -0,0 +1,20 @@
+
+  createUser( name: string, edad:number, apellido:string, telefono:number ){
+    const user = {
+      id: v4(),
+      edad,
+      name,
+      apellido,
+      telefono,
```
las variables no van mezcladas, es ingles o español

Cruz-Flores commented on src/users/user.service.ts:
```diff
@@ -0,0 +1,20 @@
+@Injectable()
+export class UserService {
+
+  private users = []
```
no esta tipado

susana-vargas committed:
se corrigieron algunos decoradores

Cruz-Flores committed:
refactor: improve code readability in main.ts and user.controller.ts

This commit refactors the `main.ts` and `user.controller.ts` files to improve code readability. The changes include better formatting and organization of the code, which makes it easier to understand and maintain.
